Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -578,6 +580,31 @@ public async Task RequestHandler(IHttpRequest httpReq, string urlString, string
             }
         }
 
+        /// <summary>
+        /// Get the default CORS headers
+        /// </summary>
+        /// <param name="req"></param>
+        /// <returns></returns>
+        public IDictionary<string, string> GetCorsHeaders(IRequest req)
+        {
+            var origin = req.Headers["Origin"];
+            if (origin == StringValues.Empty)
```

Comment:
why not
```suggestion
            if (String.IsNullOrWhiteSpace(origin))
```
?

and below, too.

Reply:
I don't believe that would be correct. `req.Headers` is a `IHeaderDictionary`, which returns `StringValues.Empty` when the value is not present. See docs [here](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.iheaderdictionary.item?view=aspnetcore-3.1#Microsoft_AspNetCore_Http_IHeaderDictionary_Item_System_String_).